src/NATS.Client.JetStream/NatsJSOpts.cs

```diff
@@ -4,19 +4,29 @@ namespace NATS.Client.JetStream;
 
 public record NatsJSOpts
 {
-    public NatsJSOpts(NatsOpts opts, string? apiPrefix = default, int? maxMsgs = default, AckOpts? ackOpts = default, string? inboxPrefix = default)
+    public NatsJSOpts(NatsOpts opts, string? apiPrefix = default, string? domain = default, AckOpts? ackOpts = default, string? inboxPrefix = default)
     {
         ApiPrefix = apiPrefix ?? "$JS.API";
-        MaxMsgs = maxMsgs ?? 1000;
         AckOpts = ackOpts ?? new AckOpts(opts.WaitUntilSent);
         InboxPrefix = inboxPrefix ?? opts.InboxPrefix;
```

Comment:
Should we get rid of `InboxPrefix` and always use the one from `NatsOpts`?  I can't see a reason why we'd need a different one for the JS Context

Reply:
Agreed.